src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -685,7 +685,15 @@ public static string ConvertPathToDocumentUri(string path)
             }
 
             // ' is not always encoded.  I've seen this in Windows PowerShell.
-            return docUriStrBld.Replace("'", "%27").Insert(0, fileUriPrefix).ToString();
+            docUriStrBld.Replace("'", "%27");
+
+            // Insert /// unless path is a UNC path in which case the proper URI form is file://server/share.
+            if ((docUriStrBld.Length < 2) || ((docUriStrBld[0] != '/') && (docUriStrBld[1] != '/')))
```

Comment:
I could use a little bit more explanation as to how this properly checks for UNC paths. One windows, local paths will start with:

```
c%3A
```
and non-Windows will start with only 1 forward slash?

Reply:
I'll update the comment but basically, a UNC path will start with `//` after the `\` to `/` conversion happens.  The proper doc uri form for a UNC path is `file://server/share/...`  So in this case, do not add any additional `///` chars.

Reply:
> On windows, local paths will start with `c%3A`

Actually they could start with any alpha char. I mention this since it's a bug in our tests (try testing a build in the D: drive).

But yes a comment describing what we're looking for would be helpful I think

Reply:
it was an example, my bad for being vague there